tests/php/src/Internal/ProductAttributesLookup/FiltererTest.php

```diff
@@ -0,0 +1,1443 @@
+<?php
+
+namespace Automattic\WooCommerce\Tests\Internal\ProductAttributesLookup;
+
+use Automattic\WooCommerce\Internal\AttributesHelper;
+use Automattic\WooCommerce\RestApi\UnitTests\Helpers\ProductHelper;
+use Automattic\WooCommerce\Utilities\ArrayUtil;
+
+/**
+ * Tests related to filtering for WC_Query.
+ */
+class FiltererTest extends \WC_Unit_Test_Case {
+
+	/**
+	 * Runs before all the tests in the class.
+	 */
+	public static function setupBeforeClass() {
+		global $wpdb, $wp_post_types;
+
+		parent::setUpBeforeClass();
+
+		$wpdb->query(
+			"
+          CREATE TABLE IF NOT EXISTS {$wpdb->prefix}wc_product_attributes_lookup (
+		  product_id bigint(20) NOT NULL,
+		  product_or_parent_id bigint(20) NOT NULL,
+		  taxonomy varchar(32) NOT NULL,
+		  term_id bigint(20) NOT NULL,
+		  is_variation_attribute tinyint(1) NOT NULL,
+		  in_stock tinyint(1) NOT NULL
+ 		  );
```

Comment:
Initial indent seems slightly off in this area.

Reply:
The phpcbf tool didn't touch it so I missed it, thanks.